src/pages/checkIn.js

```diff
@@ -42,20 +47,23 @@ export const checkInTemplate = async (data) => {
 
                         template += `</select>`;
                     }
-                        
-                template += `
-
-                </div>
-                <div class="col-md-3">
-                    <button class="btn btn-outline-primary btn-block text-nowrap" ${!isCheckedIn ? `disabled` : visitCollections && visitCollections.length > 0 ? `` : `disabled`} type="submit" id="checkInComplete">${isCheckedIn ? `Check-Out` : `Check-In`}</button>
+                    template += `
                 </div>
```

Comment:
Where is the opening div tag for this closed div tag?

Reply:
line 34